src/local-editor-service.ts

```diff
@@ -98,21 +97,45 @@ export class LocalEditorService extends EditorService {
           };
         })
       };
-    } else if (location.kind === 'attribute') {
+    }
+
+    if (location.kind === 'attributeValue') {
+      const domModules = document.getByKind('dom-module');
+      if (domModules.size > 1) return;
```

Comment:
Hm, that's annoying. Looks like parse5 is just implementing what html does though? There's no way to go from a template's document fragment to the template element?

We could do a top-down walk of the document and build a map from document fragment to its containing template. We'd have to also write custom ancestor walkers, but it's doable.

Feel free to punt this to a follow up PR though, just also file an issue to track that effort and leave a comment here with short description and a link to the issue.